test/board_test.exs

```diff
@@ -1,15 +1,20 @@
 defmodule BoardTest do
   use ExUnit.Case
 
-  test "places a mark on an empty board 2" do
+  test "returns an empty board" do
+    board = Board.empty_board
+    assert !Board.board_full?(board)
+  end
+
+  test "places mark on empty board" do
     assert Board.place_mark([1,2,3,4,5,6,7,8,9], 1) == {:ok, ["X",2,3,4,5,6,7,8,9]}
```

Comment:
Thanks, I changed it. Also had a look at all my other tests and tidied them up a bit.